main/templates/base.html

```diff
@@ -80,34 +88,27 @@
 
 	<footer>
 		<div class="container">
-			<!-- <div class="row">
-				<div class="col-sm-6">
-					<h3>Explore</h3>
-					<nav>
-						<ul>
-							<li><a class="footerButton" href="{% url 'home' %}">Home</a></li>
-							<li><a class="footerButton" href="{% url 'our_team' %}">Our Team</a></li>
-							<li><a class="footerButton" href="{% url 'projects' %}">Projects</a></li>
-							<li><a class="footerButton" href="{% url 'blog_index' %}">Blog</a></li>
-						</ul>
-					</nav>
-				</div>
-
-				<div class="col-sm-6">
-					<h3>Contact Us</h3>
-					<nav>
-						<ul>
-							<li><a class="footerButton" href="http://www.facebook.com/EnactusWaterloo">Facebook</a></li>
-							<li><a class="footerButton" href="mailto:info@enactuswaterloo.ca">Email</a></li>
-							<li><a class="footerButton" href="https://twitter.com/enactusUW">Twitter</a></li>
-						</ul>
-					</nav>
-				</div>
-
-			</div> -->
-			<div class="text-center" id="copyrightMessage">Copyright &copy; Enactus Waterloo 2015.</div>
+			<div class="text-center" id="copyrightMessage">Copyright &copy; Enactus Waterloo 2015</div>
 		</div>
 	</footer>
 
+	<script>
+		function hasHtml5Validation () {
+			return typeof document.createElement('input').checkValidity === 'function';
```

Comment:
Good job. It's annoying that we have to do this just for one major browser, hopefully Apple will catch up soon (Safari 9 won't validate inputs either, so the earliest you can expect it is 2016).